lib/settings/pages/video_player_settings.dart

```diff
@@ -0,0 +1,188 @@
+              child: Column(
+                children: [
+                  ToggleOption(
+                    description: l10n.videoAutoFullscreen,
+                    // subtitle: l10n.videoAutoFullscreen,
+                    value: videoAutoFullscreen,
+                    iconEnabled: Icons.fullscreen,
+                    iconDisabled: Icons.fullscreen_exit,
+                    onToggle: (bool value) => setPreferences(LocalSettings.videoAutoFullscreen, value),
+                    highlightKey: settingToHighlight == LocalSettings.videoAutoFullscreen ? settingToHighlightKey : null,
+                  ),
+                  ToggleOption(
+                    description: l10n.videoAutoMute,
+                    //subtitle: l10n.sidebarBottomNavDoubleTapDescription,
+                    value: videoAutoMute,
+                    iconEnabled: Icons.volume_off,
+                    iconDisabled: Icons.volume_up,
+                    onToggle: (bool value) => setPreferences(LocalSettings.videoAutoMute, value),
+                    highlightKey: settingToHighlight == LocalSettings.videoAutoMute ? settingToHighlightKey : null,
+                  ),
+                  ToggleOption(
+                    description: l10n.videoAutoLoop,
+                    subtitle: l10n.videoAutoLoop,
+                    value: videoAutoLoop,
+                    iconEnabled: Icons.loop,
+                    iconDisabled: Icons.loop_outlined,
+                    onToggle: (bool value) => setPreferences(LocalSettings.videoAutoLoop, value),
+                    highlightKey: settingToHighlight == LocalSettings.videoAutoLoop ? settingToHighlightKey : null,
+                  ),
+                  ListOption(
+                    description: l10n.videoAutoPlay,
+                    value: ListPickerItem(label: videoAutoPlay.name, icon: Icons.video_settings_outlined, payload: videoAutoPlay),
+                    options: [
+                      ListPickerItem(label: VideoAutoPlay.never.label, payload: VideoAutoPlay.never),
+                      ListPickerItem(label: VideoAutoPlay.always.label, payload: VideoAutoPlay.always),
+                      ListPickerItem(label: VideoAutoPlay.onWifi.label, payload: VideoAutoPlay.onWifi),
+                    ],
+                    icon: Icons.play_circle,
+                    onChanged: (value) async => setPreferences(LocalSettings.videoAutoPlay, value.payload.name),
+                    highlightKey: settingToHighlight == LocalSettings.videoAutoPlay ? settingToHighlightKey : null,
+                  ),
+                  ListOption(
+                    description: l10n.videoDefaultPlaybackSpeed,
+                    value: ListPickerItem(label: videoDefaultPlaybackSpeed.label, icon: Icons.speed, payload: videoDefaultPlaybackSpeed),
+                    options: [
+                      ListPickerItem(label: VideoPlayBackSpeed.pointTow5x.label, payload: VideoPlayBackSpeed.pointTow5x),
```

Comment:
Can you try to add icons here too? Even if they're all the same? I think it looks better in the bottom modal.